cabalscan/src/CabalScan/Options.hs

```diff
@@ -47,5 +47,5 @@ printMsgAndExit = \case
                        "-h,--help                Show this help text"]
     missing = unlines ["Missing: CABAL_FILES..."]
     wrongpath p = unlines ["Couldn't parse absolute file path: " ++ p]
-    putErrLn :: [String] -> IO ()
-    putErrLn = hPutStrLn stderr . intercalate "\n"
+    hPutLn :: Handle -> [String] -> IO ()
+    hPutLn h = hPutStrLn h . intercalate "\n"
```

Comment:
You might want to use `unlines` here too.

Reply:
Yes. This has been fixed. Somehow in this way: `hPutStrLn h . intercalate "\n" . map unlines`.

BTW I need `intercalate` to separate messages with a blank line so they look more less like that:
```
Missing: CABAL_FILES...                                     << here is where unlines does the job
                                                                              << here is where 'intercalate \n' does the job
Usage: cabalscan CABAL_FILES...
  Prints in stdout information extracted from Cabal files in JSON format.

```

Reply:
> `hPutStrLn h . intercalate "\n" . map unlines`

How about `hPutStrLn h . unlines . map unlines`?

Reply:
> How about hPutStrLn h . unlines . map unlines?

`unlines` adds additional space and the end of the array while intercalate is not. In this case, such expression generates double blank line at the end of the message:
```
cabalscan - extract build information from Cabal files

Usage: cabalscan CABAL_FILES...
  Prints in stdout information extracted from Cabal files in JSON format.

Available options:
-h,--help                Show this help text


```